Binary files source/.git/index and source-new/.git/index differ
diff '--color=auto' -rupwN source/Makefile source-new/Makefile
--- source/Makefile	2024-10-21 02:46:06.720225834 -0400
+++ source-new/Makefile	2024-10-25 01:03:37.283351544 -0400
@@ -5,10 +5,11 @@ ifneq ($(filter Msys Cygwin, $(shell una
     TYRIAN_DIR = C:\\TYRIAN
 else
     PLATFORM := UNIX
-    TYRIAN_DIR = $(gamesdir)/tyrian
+    TYRIAN_DIR ?= $(gamesdir)/tyrian
 endif
 
-WITH_NETWORK := true
+WITH_NETWORK ?= true
+REDOX_OVERRIDE ?= false
 
 ################################################################################
 
@@ -114,11 +115,15 @@ installdirs :
 	mkdir -p $(DESTDIR)$(docdir)
 	mkdir -p $(DESTDIR)$(man6dir)
 	mkdir -p $(DESTDIR)$(desktopdir)
-	mkdir -p $(DESTDIR)$(icondir)/hicolor/22x22/apps
-	mkdir -p $(DESTDIR)$(icondir)/hicolor/24x24/apps
-	mkdir -p $(DESTDIR)$(icondir)/hicolor/32x32/apps
-	mkdir -p $(DESTDIR)$(icondir)/hicolor/48x48/apps
-	mkdir -p $(DESTDIR)$(icondir)/hicolor/128x128/apps
+	if [ "$(REDOX_OVERRIDE)" = "true" ]; then\
+		mkdir -p $(DESTDIR)$(icondir);\
+	else\
+		mkdir -p $(DESTDIR)$(icondir)/hicolor/22x22/apps;\
+		mkdir -p $(DESTDIR)$(icondir)/hicolor/24x24/apps;\
+		mkdir -p $(DESTDIR)$(icondir)/hicolor/32x32/apps;\
+		mkdir -p $(DESTDIR)$(icondir)/hicolor/48x48/apps;\
+		mkdir -p $(DESTDIR)$(icondir)/hicolor/128x128/apps;\
+	fi;\
 
 .PHONY : install
 install : $(TARGET) installdirs
@@ -126,11 +131,15 @@ install : $(TARGET) installdirs
 	$(INSTALL_DATA) NEWS README $(DESTDIR)$(docdir)/
 	$(INSTALL_DATA) linux/man/opentyrian.6 $(DESTDIR)$(man6dir)/opentyrian$(man6ext)
 	$(INSTALL_DATA) linux/opentyrian.desktop $(DESTDIR)$(desktopdir)/
-	$(INSTALL_DATA) linux/icons/tyrian-22.png $(DESTDIR)$(icondir)/hicolor/22x22/apps/opentyrian.png
-	$(INSTALL_DATA) linux/icons/tyrian-24.png $(DESTDIR)$(icondir)/hicolor/24x24/apps/opentyrian.png
-	$(INSTALL_DATA) linux/icons/tyrian-32.png $(DESTDIR)$(icondir)/hicolor/32x32/apps/opentyrian.png
-	$(INSTALL_DATA) linux/icons/tyrian-48.png $(DESTDIR)$(icondir)/hicolor/48x48/apps/opentyrian.png
-	$(INSTALL_DATA) linux/icons/tyrian-128.png $(DESTDIR)$(icondir)/hicolor/128x128/apps/opentyrian.png
+	if [ "$(REDOX_OVERRIDE)" = "true" ]; then\
+		$(INSTALL_DATA) linux/icons/tyrian-32.png $(DESTDIR)$(icondir)/opentyrian.png;\
+	else\
+		$(INSTALL_DATA) linux/icons/tyrian-22.png $(DESTDIR)$(icondir)/hicolor/22x22/apps/opentyrian.png;\
+		$(INSTALL_DATA) linux/icons/tyrian-24.png $(DESTDIR)$(icondir)/hicolor/24x24/apps/opentyrian.png;\
+		$(INSTALL_DATA) linux/icons/tyrian-32.png $(DESTDIR)$(icondir)/hicolor/32x32/apps/opentyrian.png;\
+		$(INSTALL_DATA) linux/icons/tyrian-48.png $(DESTDIR)$(icondir)/hicolor/48x48/apps/opentyrian.png;\
+		$(INSTALL_DATA) linux/icons/tyrian-128.png $(DESTDIR)$(icondir)/hicolor/128x128/apps/opentyrian.png;\
+	fi;\
 
 .PHONY : uninstall
 uninstall :
